app/src/main/java/io/graphqlcrud/app/GraphQLResource.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2012-2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.graphqlcrud.app;
+
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import graphql.ExecutionInput;
+import graphql.ExecutionResult;
+import graphql.GraphQL;
+import graphql.execution.instrumentation.ChainedInstrumentation;
+import graphql.execution.instrumentation.ExecutionStrategyInstrumentationContext;
+import graphql.execution.instrumentation.Instrumentation;
+import graphql.execution.instrumentation.SimpleInstrumentation;
+import graphql.execution.instrumentation.parameters.InstrumentationExecutionStrategyParameters;
+import graphql.execution.instrumentation.tracing.TracingInstrumentation;
+import graphql.schema.GraphQLSchema;
+import graphql.schema.idl.SchemaPrinter;
+import io.agroal.api.AgroalDataSource;
+import io.graphqlcrud.DatabaseSchemaBuilder;
+import io.graphqlcrud.GraphQLSchemaBuilder;
+import io.graphqlcrud.SQLContext;
+import io.graphqlcrud.model.Schema;
+
+@Path("/graphql")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class GraphQLResource {
+    
+    @Inject
+    private AgroalDataSource datasource;
+    
+    private GraphQLSchema schema;
+    
+    @ConfigProperty(name = "quarkus.datasource.schema")
```

Comment:
```suggestion
    @ConfigProperty(name = "app.database.schema")
```

`quarkus.datasource.schema` is not a recognised valid quarkus configuration knob, this will print warning. 

A very minor FYI.